package.json

```diff
@@ -16,28 +16,30 @@
     "gmaps",
     "angular2"
   ],
+  "packageScope": "@agm",
+  "packages": [
+    "packages/core",
+    "packages/js-marker-clusterer",
+    "packages/snazzy-info-window"
+  ],
   "scripts": {
-    "clean": "rimraf dist && rimraf .tmp && mkdir dist && mkdir .tmp && rimraf 'packages/**/*.ngfactory.ts'",
-    "prebuild": "npm run clean",
-    "build": "npm run lint && npm run scripts && npm run packagejson && npm run copyassets",
+    "clean": "rimraf dist rimraf node_modules/@agm rimraf .tmp 'packages/**/*.ngfactory.ts' && mkdirp dist .tmp",
```

Comment:
shouldn't there be an `&&` between dist and rimraf, otherwise you're trying to delete the rimraf directory.

Reply:
I'm not sure why I left `rimraf` in there. `rimraf` can take multiple arguments and I didn't want any `&&`'s but I didn't need to leave the extra `rimraf`'s in there